tidypolars/tibble.py

```diff
@@ -174,7 +177,7 @@ def count(self, *args, sort = False, name = 'n'):
 
         return out
 
-    def distinct(self, *args):
+    def distinct(self, *args, **kwargs):
```

Comment:
It doesn't look like the function below uses `kwargs` except to grab `keep_all`.  and in R `distinct()` uses `.keep_all` (with the period).

Can you redefine with this instead?
```python
def distinct(self, *args, _keep_all = False)
```

Reply:
R has many arguments starting with dot. I would just remove that "dot" and use keep_all.  _keep_all is weird, and add one unnecessary character every time one needs to type it.

For the function signature, makes sense.

Reply:
The reason for the `.keep_all` in R and the `_keep_all` in python is so that you can use kwargs but still use a common name.

In R you can do something like this and rename a column on the fly.
```r
df %>%
    distinct(keep_all = y, .keep_all = TRUE)
```

Any functions where `kwargs` might be used in this fashion we should have an underscore even though it adds typing to the arg.

Reply:
I still think we should use keep_all instead of _keep_all. The reasons is that we are not using any R API, like tpy2. We are doing it natively in python, so the arguments will not be translated.  I saw other packages taking that approach, and it makes sense to me. I will use keep_all for now, but we can discuss it latter.

Reply:
The reason R does it is to help with kwargs and it helps the same in python. Another good example is `mutate()` and the `.before` arg. In the future we need to add `_before` to `.mutate()`

```r
df %>%
    mutate(before = x + 1, .before = y)
```

Reply:
This is also the same approach that `siuba` takes when porting `pandas` syntax to `dplyr`

https://siuba.org/api/distinct